packages/common/lib/types/OpenID4VCIServerMetadata.ts

```diff
@@ -1,7 +1,87 @@
-import { CredentialFormat } from '@sphereon/ssi-types';
+import { W3CVerifiableCredential } from '@sphereon/ssi-types';
 
+import {CredentialOffer} from "./CredentialIssuance.types";
 import { OAuth2ASMetadata } from './OAuth2ASMetadata';
 
+export interface IssuerMetadata {
+}
+
+export interface CredentialOfferWithBaseURL extends CredentialOffer {
```

Comment:
The keys are different ... almost all of the properties are different.
Since we are using all statics ... we can not benefit from hirarchy. I have removed the base class 'CredentialOffer'.